src/main/java/org/privacyidea/AuthenticationStatus.java

```diff
@@ -0,0 +1,8 @@
+package org.privacyidea;
+
+public enum AuthenticationStatus
+{
+    CHALLENGE,
```

Comment:
Probably need a default case (like "NONE") for initalization

Reply:
fixed